package.js

```diff
@@ -1,66 +1,64 @@
 Package.describe({
-    name: 'universe:i18n',
-    documentation: './atmosphere.md',
-    version: '1.31.0',
-    summary: 'Lightweight i18n, YAML & JSON translation files, React component, incremental & remote loading',
-    git: 'https://github.com/vazco/meteor-universe-i18n'
+  name: 'universe:i18n',
+  documentation: './atmosphere.md',
+  version: '1.31.0',
+  summary:
+    'Lightweight i18n, YAML & JSON translation files, React component, incremental & remote loading',
+  git: 'https://github.com/vazco/meteor-universe-i18n',
 });
 
-var npmDependencies = {
-    'strip-json-comments': '3.1.1',
-    'js-yaml': '3.14.0'
+const npmDependencies = {
+  'js-yaml': '4.1.0',
+  'strip-json-comments': '3.1.1',
 };
 
+Npm.depends(npmDependencies);
 Package.registerBuildPlugin({
-    name: 'UniverseI18n',
-    use: ['ecmascript', 'caching-compiler@1.2.2'],
-    sources: ['builder.js', 'lib/utilities.js'],
-    npmDependencies: npmDependencies
+  name: 'universe:i18n',
+  use: ['caching-compiler@1.2.2', 'tracker', 'typescript'],
+  sources: [
+    'source/common.ts',
+    'source/compiler.ts',
+    'source/locales.ts',
+    'source/utils.ts',
+  ],
+  npmDependencies,
 });
 
-Npm.depends(npmDependencies);
-
 Package.onUse(function (api) {
-    api.versionsFrom('1.6');
-
-    api.use([
-        'ddp',
-        'fetch@0.1.1',
-        'check',
-        'webapp',
-        'tracker',
-        'promise',
-        'ecmascript',
-        'isobuild:compiler-plugin@1.0.0'
-    ]);
+  api.versionsFrom('2.0');
```

Comment:
Does this field relate to the package's version? If so, do we want to call it a major release even though there should be no breaking changes? Or is it not semver?

Reply:
I _think_ it'll actually work with 1.6, but we'd need to check that.

Reply:
This package cannot be installed in the project with the 1.6 version of Meteor. There are several package version conflicts.